src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -110,11 +118,16 @@ const WorkflowDashboard = _.flow(
     }), simplifiedFailures)
   }
 
+  const callNames = _.keys(calls)
```

Comment:
"One day" ™️  it would be nice to sort by start time instead of alphabetically, but I agree that alphabetically is infinitely better than random for now.

Reply:
Why not use [`_.orderBy`](https://lodash.com/docs/4.17.15#orderBy) when you save `calls` into state?

Reply:
(that would mean you had to convert `calls` to an array, so that might be way out of scope...)